public/styles.css

```diff
@@ -1,229 +1,226 @@
 body {
-    background: linear-gradient(45deg, #415A85, #415A85);
-    margin: 0;
+  background: linear-gradient(45deg, #415A85, #415A85);
+  margin: 0;
 }
```

Comment:
Sorry, had to fix the tab size in this file. Not sure how it ended up being 2-space tabs.